BLSQ/iaso
# IA-2916: Have mobile feature flags available in French

Explain what problem this PR is resolving
- Have mobile feature flags available in French
Related JIRA tickets : [IA-2916](https://bluesquare.atlassian.net/browse/IA-2916)

## Self proofreading checklist


## Changes
- Add french translations for featureFlags

## How to test
- Go in projects
- Change the language to french
- Check if featureflags are in french

## Print screen / video
![Screenshot from 2024-06-08 13-11-17](https://github.com/BLSQ/iaso/assets/19631540/a11b196b-670a-4f04-815f-5a96be592c22)
![Screenshot from 2024-06-08 13-11-48](https://github.com/BLSQ/iaso/assets/19631540/65be25ee-91c4-4124-8fc9-de660126e49a)



[IA-2916]: https://bluesquare.atlassian.net/browse/IA-2916?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
